build/copy-files-for-packaging.cmd

```diff
@@ -0,0 +1,32 @@
+@echo ==========================
+@echo Copying source files for NuGet symbols package
+xcopy /ys src\* %TO_PACK_TARGET%\src\src\
+
+@echo ==========================
+@echo Copying signed files to the NuGet lib path
+set PORTABLE_LIB=portable-net45+win
+
+md %TO_PACK_TARGET%\lib\%PORTABLE_LIB%\
+copy /y  %SIGNED%\*.dll "%NUGET_TARGET%\lib\%PORTABLE_LIB%\"
+copy /y %TO_SIGN_TARGET%\*.pdb "%NUGET_TARGET%\lib\%PORTABLE_LIB%\"
+copy /y %TO_SIGN_TARGET%\*.xml "%NUGET_TARGET%\lib\%PORTABLE_LIB%\"
+
+call %COPY_TO_NUGET_LIBFOLDER% Desktop net45
```

Comment:
It's failing, amongst those issues, with errors relating to the unset and missing `%COPY_TO_NUGET_LIBFOLDER%`
```
2017-07-07T13:59:41.0363393Z C:\vsts-agent\_work\7\s>call  WinRT netcore45 
2017-07-07T13:59:41.0383416Z ##[error]'WinRT' is not recognized as an internal or external command,
operable program or batch file.